test/unit/language-specs.js

```diff
@@ -0,0 +1,106 @@
+import sinon from 'sinon';
+import chai from 'chai';
+import XCUITestDriver from '../..';
+
+chai.should();
+
+describe('language and locale', () => {
+  const LANGUAGE = "en";
+  const LOCALE = "en_US";
+
+  describe('send only language and locale', () => {
+    it('should send translated POST /session request with valid desired caps to WDA', async () => {
+      let args = [];
+      args.push('-AppleLanguages');
+      args.push(`(${LANGUAGE})`);
+      args.push('-NSLanguages');
+      args.push(`(${LANGUAGE})`);
+
+      args.push('-AppleLocale');
+      args.push(LOCALE);
+
+      let desired = {
```

Comment:
I would rather this be named something like `expectedWDACapabilities`. As it stands it is a bit difficult to parse what is happening.